environment.yml

```diff
@@ -4,8 +4,10 @@ channels:
   - defaults
 dependencies:
   - astropy
+  - black
   - filelock
   - inflect
+  - isort
```

Comment:
What about pylint?

Reply:
I thought that black and isort were sort of same same but different. Now I know better when running pylint. :)